### PR TITLE
update changelogs and fix text in resource.

### PR DESCRIPTION
**Pull request by sumitgupta3**
We found the changelogs were missing for Edit properties of a message launch. 
- added changelogs 
- fixed text in chat message resource.

Edit Properties Launch commit : https://github.com/microsoftgraph/microsoft-graph-docs/pull/16836 
Please advise if changelog changes should be only in Microsoft.Teams.GraphSvc.json Or also in Microsoft.Teams.Core.json

### CODE REVIEW

File: changelog/Microsoft.Teams.Core.json [hunk 1]
```diff
@@ -5541,6 +5541,24 @@
       "CreatedDateTime": "2022-08-22T15:04:69.427Z",
       "WorkloadArea": "Teamwork",
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "1e139b6c-35c7-4d54-8d37-cc2e812829db7",
+          "ApiChange": "Method",
+          "ChangedApiName": "chat message",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:5550]: If this changelog entry pertains to the addition of an operation, use the HTTP verb here, i.e. PATCH.

File: changelog/Microsoft.Teams.Core.json [hunk 2]
```diff
@@ -5541,6 +5541,24 @@
...
+          "ChangeType": "Update",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:5551]: This is not one of the possible values. If you're documenting the addition of an Update method, the value here would be "Addition".

File: api-reference/beta/resources/chatmessage.md [hunk 3]
```diff
@@ -27,14 +27,14 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |[Create subscription for new channel messages](../api/subscription-post-subscriptions.md) | [subscription](subscription.md) | Listen for new, edited, and deleted messages, and reactions to them. |
 |[Get message in channel](../api/chatmessage-get.md) | [chatMessage](chatmessage.md) | Get a single root message in a channel.|
 |[Send message in channel](../api/chatmessage-post.md) | [chatMessage](chatmessage.md)| Create a new root message in a channel.|
-|[Update message in channel](../api/chatmessage-update.md)|[chatMessage](chatmessage.md)| Update the **policyViolation** property of a chat message.|
+|[Update message in channel](../api/chatmessage-update.md)|[chatMessage](chatmessage.md)| Update the properties of a chat message.|
```
Comment (Lauragra) [inline api-reference/beta/resources/chatmessage.md:30]: Is this just a correction to the documentation or a change to the behavior of the API? If the former, you don't need a changelog entry for the update.

File: changelog/Microsoft.Teams.Core.json [hunk 4]
```diff
@@ -5541,6 +5541,24 @@
...
+          "Description": "Added support for the updation of properties of the [chatMessage](https://docs.microsoft.com/en-us/graph/api/resources/chatMessage?view=graph-rest-beta) resource.",
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.Core.json:5552]: I'm not sure from the description what the intent of this entry is. We might not need a changelog entry if all you're doing is clarifying the method descriptions in the resource topic. Is this an actual change in the API behavior?

File: changelog/Microsoft.Teams.GraphSvc.json [hunk 5]
```diff
@@ -617,6 +617,24 @@
       "CreatedDateTime": "2022-07-18T11:01:30.771Z",
       "WorkloadArea": "Teamwork",
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
```
Comment (Lauragra) [inline changelog/Microsoft.Teams.GraphSvc.json:622]: You only need to add the changelog entry to one changelog file so please remove this one, assuming we need a changelog entry at all.

File: api-reference/beta/resources/chatmessage.md (hunk 3 above)
Comment (sumitgupta3) [reply api-reference/beta/resources/chatmessage.md:30]: When we introduce this change, there was no mention of the same in changelogs. So now, if any developer goes to changelogs he will never the change in changelogs. That's why I have added it now after talking to my PM.

There was some issue with docs for update message as highlighted above so I have corrected that as well.
And it's causing confusion as well. 

For v1.0, not launched yet we will take care during launch.

File: changelog/Microsoft.Teams.Core.json (hunk 1 above)
Comment (sumitgupta3) [reply changelog/Microsoft.Teams.Core.json:5550]: where I need to use PATCH is not clear, please share.

File: changelog/Microsoft.Teams.Core.json (hunk 2 above)
Comment (sumitgupta3) [reply changelog/Microsoft.Teams.Core.json:5551]: Its updation of Update method as it was there previously and now we support updation with more fields.

File: changelog/Microsoft.Teams.Core.json (hunk 4 above)
Comment (sumitgupta3) [reply changelog/Microsoft.Teams.Core.json:5552]: Yes, this is actual change. 
Update Message previously supported only PolicyViolation update in Application context.
Now, Update Message supports all the properties update in user context.

original one here: https://github.com/microsoftgraph/microsoft-graph-docs/pull/16836

File: changelog/Microsoft.Teams.GraphSvc.json (hunk 5 above)
Comment (sumitgupta3) [reply changelog/Microsoft.Teams.GraphSvc.json]: removed

**Review by Lauragra — APPROVED** [review]

**Pull request merged**